test/api/v3/integration/groups/POST-groups_invite.test.js

```diff
@@ -0,0 +1,198 @@
+import {
+  generateUser,
+  translate as t,
+} from '../../../../helpers/api-integration.helper';
+
+describe('Post /groups/:groupId/invite', () => {
+  let user;
+  let group;
+
+  beforeEach(async () => {
+    user = await generateUser({balance: 1});
+    let groupName = 'Test Public Guild';
+    let groupType = 'guild';
+    group = await user.post('/groups', {
+      name: groupName,
+      type: groupType,
+    });
+  });
+
+  describe('user id invites', () => {
+    it('returns an error when invited user is not found', async () => {
+      let fakeID = 'fakeID';
+
+      await expect(user.post(`/groups/${group._id}/invite`, {
+        uuids: [fakeID],
+      }))
+      .to.eventually.be.rejected.and.eql({
+        code: 404,
+        error: 'NotFound',
+        message: t('userWithIDNotFound', {userId: fakeID}),
+      });
+    });
+
+    it('invites a user to a group by uuid', async () => {
+      let newUser = await generateUser();
+      let invite = await user.post(`/groups/${group._id}/invite`, {
+        uuids: [newUser._id],
+      });
+      let invitedUser = await newUser.get('/user');
```

Comment:
This could instead be:

```
await expect(invitedUser.get('/user')).to.eventually.have.deep.property('invitedUser.invitations.guilds[0].id', group._id);
```